src/missive.js

```diff
@@ -1,4 +1,7 @@
-const { openai } = require("./openai");
+const { JSDOM } = require("jsdom");
+const { supabaseTachio } = require("./supabaseclient");
+const DAILY_REPORT_REGEX = /Hi.*What has the team done since the last call\/email regarding this project\??(.*)What will the team do between now and the next call\/email regarding this project\??(.*)What impedes the team from performing their work as effectively as possible\??(.*)How much time have we spent today\??(.*)How much time have we spent this week\??(.*)How much time have we spent this month\??(.*)Our team today:?(.*)Regards/;
```

Comment:
- Can you please add a separate section for design with the following heading: "What has the design team done today?"
- We should extract the `[Design] # billable hour(s)` as a separate section after `How much time have we spent today/this week/this month?`
<img width="688" alt="Screenshot 2024-04-12 at 4 42 39 PM" src="https://github.com/PublicDataWorks/tachio/assets/4379424/76dc9e38-0237-49a3-8832-a5ec06bc2fb3">
- Please do not extract the sections: 
-`How much time have we spent this week` 
- `How much time have we spent this month`
<img width="335" alt="Screenshot 2024-04-12 at 4 45 02 PM" src="https://github.com/PublicDataWorks/tachio/assets/4379424/b2861b1f-4a20-4b83-9443-955d31f04fa6">

Reply:
`We should extract the [Design] # billable hour(s) as a separate section after How much time have we spent today/this week/this month?`

Based on this, are you suggesting to create a separate column for `design`, or should we use a single array column?

Reply:
I think we should create a separate column for `design`.

Reply:
It's been done.